### PR TITLE
Extensions

### PR DESCRIPTION
**Pull request by kshitijSharma2014**

### CODE REVIEW

**Comment by kshitijSharma2014** [comment]
@puneet0191  @javigomez  @gunjanpatel 
added doc blocks

File: tests/system/webdriver/tests/extensions/ModuleManager0002Test.php [hunk 1]
```diff
@@ -23,18 +23,6 @@ class ModuleManager0001Test extends JoomlaWebdriverTestCase
...
-	 * @since   3.0
-	 *
-	 * @return void
-	 */
-	public function setUp()
-	{
-		parent::setUp();
-	}
-
-	/**
```
Comment (puneet0191) [inline tests/system/webdriver/tests/extensions/ModuleManager0002Test.php:37]: Why Did you remove this?

File: tests/system/webdriver/tests/extensions/ModuleManager0002Test.php [hunk 2]
```diff
@@ -108,7 +99,6 @@ public function addModule_ArticleCategories_ModuleAdded()
 
 		$this->doAdminLogin();
 		$this->driver->get($cfg->host . $cfg->path . $categoryManager);
-		$this->categoryManagerPage = $this->getPageObject('CategoryManagerPage');
```
Comment (puneet0191) [inline tests/system/webdriver/tests/extensions/ModuleManager0002Test.php:111]: Why did you remove this?

File: tests/system/webdriver/tests/extensions/ModuleManager0002Test.php [hunk 3]
```diff
@@ -202,17 +193,15 @@ public function addModule_ArticleCategories_ModuleAdded_2()
 		$this->assertTrue($this->siteHomePage->itemExist($articleName1, 'a'));
 		$this->siteHomePage->itemClick($categoryName2);
 		$this->assertTrue($this->siteHomePage->itemExist($articleName2, 'a'));
-		
 		$this->doAdminLogin();
+
 		$this->driver->get($cfg->host . $cfg->path . $articleManager);
-		$this->articleManagerPage = $this->getPageObject('ArticleManagerPage');
```
Comment (puneet0191) [inline tests/system/webdriver/tests/extensions/ModuleManager0002Test.php:208]: Same!

**Comment by kshitijSharma2014** [comment]
@puneet0191  I am using this style of putting admin login in each test because I needed the variable $cpPage which is equal to $this->doAdminLogin(). I tried to put it in the setup but then phpcs gives me an error stating undefined variable $cpPage in the test function.

**Comment by kshitijSharma2014** [comment]
@puneet0191  @javigomez  @gunjanpatel 
updated code to meet joomla coding standards
